packages/xstate-react/test/types.test.tsx

```diff
@@ -114,4 +110,18 @@ describe('useMachine', () => {
 
     render(<YesNo />);
   });
+
+  it('state should not become never after checking state with matches', () => {
+    const YesNo = () => {
+      const [state] = useMachine<any, any>(yesNoMachine);
```

Comment:
Can you remove the `<any, any>` generics? They're not necessary

Reply:
It was the only way to reproduce the state becoming `never`. Everything seems to work fine in the test when removing the type arguments. I'm also not using `<any, any>` in my code but am running into the same problem.